app/controllers/games_controller.rb

```diff
@@ -34,8 +34,13 @@ def game_params
     params.require(:game).permit(:name)
   end
 
-  helper_method :current_game
+  helper_method :current_game, :add_player
   def current_game
     @current_game = Game.find(params[:id])
   end
+  
+  def add_player(game)
```

Comment:
Instead of adding a new action. You could make use of the existing `update` action. And pass it `black_player_id: current_user.id`